src/annotation.ts

```diff
@@ -68,8 +68,7 @@ export class Annotation extends Modifier {
   // Use the same padding for annotations as note head so the
   // words don't run into each other.
   static get minAnnotationPadding(): number {
-    const musicFont = Tables.currentMusicFont();
-    return musicFont.lookupMetric('noteHead.minPadding');
+    return Tables.lookupMetric('NoteHead.minPadding');
```

Comment:
I noticed that the metric keys now have their first letter capitalized. I guess it makes the first part look more like a class name?